.github/workflows/test.yml

```diff
@@ -53,6 +53,9 @@ jobs:
       matrix:
         ubuntu_version: ["24.04"]
         python_version: ["3.13"]
+        container_cli:
+          - docker
+          - podman
```

Comment:
```suggestion
```
This doubles the number of tests in the matrix, which means PR runs will take even longer, and the chance of a failure due to flakiness doubles.

I don't think we need to run all tests with Podman, just a subset, which we can do by adding them to the `include:` section below